gcloud/pubsub/test_subscription.py

```diff
@@ -620,29 +603,68 @@ def test_check_iam_permissions_w_alternate_client(self):
 
         self.assertEqual(len(allowed), 0)
         self.assertEqual(len(conn1._requested), 0)
-        self.assertEqual(len(conn2._requested), 1)
-        req = conn2._requested[0]
-        self.assertEqual(req['method'], 'POST')
-        self.assertEqual(req['path'], PATH)
-        self.assertEqual(req['data'], REQUESTED)
+        self.assertEqual(len(conn2._requested), 0)
+        self.assertEqual(conn2._tested_iam_permissions,
+                         (self.SUB_PATH, ROLES))
 
 
-class _Connection(object):
+class _Connection(object):  # pylint: disable=too-many-instance-attributes
```

Comment:
Yes I'm aware it is block-scopes. If you recall, way back in September 2014 we went out of our way to put all disables in our config instead of sprinkling it through our code. This is that same philosophy, so that the work then was not lost. It was your urging that put us down that path if you recall.